invrec_deutanal.py

```diff
@@ -1,304 +0,0 @@
-print("Running")
-from pyspecdata import *
-from pyspecdata.load_files.bruker_nmr import bruker_data
-from scipy.optimize import minimize,curve_fit,least_squares
-from numpy import random
```

Comment:
add this file in a new pull request

Reply:
This file is renamed and updated to proc_IR_noecho.py we can delete this